perceval/backends/core/gitlab.py

```diff
@@ -95,7 +99,7 @@ def __init__(self, owner=None, repository=None, api_token=None,
                  is_oauth_token=False, base_url=None, tag=None, archive=None,
                  sleep_for_rate=False, min_rate_to_sleep=MIN_RATE_LIMIT,
                  max_retries=MAX_RETRIES, sleep_time=DEFAULT_SLEEP_TIME,
-                 blacklist_ids=None):
+                 blacklist_ids=None, extra_retry_after_status=None):
```

Comment:
If I understand it correctly, None in this parameter means "default value". If that's the case, code would be cleaner if we have `extra_retry_after_status=[500,502]`, which would also be more explicit, or maybe `extra_retry_after_status=`DEFAULT_RETRY_AFTER_STATUS_CODES` as parameter.

Reply:
Thank you for the review @jgbarah , we generally avoid passing mutable arguments. I would merge it as it is